auth-api/tests/unit/models/test_documents.py

```diff
@@ -36,7 +36,7 @@ def test_documents_with_insert_some_type(session):
     """
     html_content = '<HTML>'
     # putting higher numbers so that version number doesnt collide with existing in db
-    doc = Documents(version_id=20, type='sometype', content=html_content)
+    doc = Documents(version_id=20, type='sometype', content=html_content, content_type='text/html')
```

Comment:
is this also needed?

Reply:
Yes, at some point someone made the content type as mandatory in model, but the change was not in DB. Now all the model is sync'ed with DB, so we need this change.